packages/www/pages/verify.jsx

```diff
@@ -14,9 +14,28 @@ import Guides from "components/Marketing/Guides";
 const Verify = () => {
   useLoggedIn();
   const router = useRouter();
-  const { user } = useApi();
+  const { user, verify } = useApi();
   const { email, emailValidToken } = router.query;
 
+  useEffect(() => {
+    if (email && emailValidToken) {
+      verify(email, emailValidToken).then(() => {
```

Comment:
Does `verify` throw an exception in case verification fails? (e.g. invalid token)
What would happen on that case? Any easy way to display an error, maybe with a toast or sth?

No problem if there's currently no pattern for that in the dashboard and it's harder than it should be (I know we have a current tech debt for not showing errors to the user on most ops).

Reply:
in case verification fails, does the user need to signup again?

Reply:
so we need to display the text like "We got an invalid token, you need to [signup](https://livepeer.com/) again"?

Reply:
in case verification fails, added the text like "Email verification failed: ${err}. Please access the llink you received by email again."